tools/start_server.py

```diff
@@ -191,26 +236,37 @@ def getch():
 
         return ch
 
-def startServer(server):
-	print("HMI signals listener was started\r")
+def start_signals_listener(server):
 	server.set_fn_new_client(new_client)
 	server.set_fn_client_left(client_left)
 	server.set_fn_message_received(message_received)
 	server.run_forever()
 
-def keyBoardEvent():
-	global server
-	char = ' '
-	while char != 'q':
-		char = getch()
+def start_file_server(file_server):
+	file_server.serve_forever()
+
+def signal_handler(sig, frame):
+    print('\rStopping server...')
+
+web_dir = os.path.dirname(os.path.dirname(__file__))
+file_server = HTTPServer(web_dir, ("", FILESERVER_PORT))
+fileServerThread = Thread(target = start_file_server, args = (file_server, ))
+fileServerThread.start()
+print("HTTP file server was started\r")
 
-server = WebsocketServer(8081)
-serverThread = Thread(target = startServer, args = (server, ))
-keyBoardThread = Thread(target = keyBoardEvent)
-keyBoardThread.start()
+server = WebsocketServer(WEBSOCKET_PORT)
+serverThread = Thread(target = start_signals_listener, args = (server, ))
 serverThread.start()
+print("HMI signals listener was started\r")
 
-keyBoardThread.join()
-print("Closing server...")
+signal.signal(signal.SIGINT, signal_handler)
+print('Press Ctrl+C to stop server')
```

Comment:
```suggestion
print("Press Ctrl+C to stop server")
```
only 2 places use single quotes

Reply:
@iCollin fixed in https://github.com/smartdevicelink/sdl_hmi/pull/274/commits/b506afc2d807d2ffa04860513272c8f6129f7f03